src/responder/kcm/kcmsrv_ops.c

```diff
@@ -1139,6 +1211,14 @@ static void kcm_op_get_cred_uuid_list_getbyname_done(struct tevent_req *subreq)
         return;
     }
 
+    ret = kcm_creds_to_table(conn_data, kcm_cc_get_cred(cc), &conn_data->creds);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_OP_FAILURE, "Unable to build credentials hash table "
+              "[%d]: %s\n", ret, sss_strerror(ret));
+        tevent_req_error(req, ret);
+        return;
+    }
+
     if (cc == NULL) {
         DEBUG(SSSDBG_MINOR_FAILURE, "No credentials by that UUID\n");
```

Comment:
1) shall not this check happen before `kcm_creds_to_table()` uses `cc`?
2) shall not message say "by that name" instead of "by that UUID"?